services/models/pod.go

```diff
@@ -93,3 +93,17 @@ func lookupImage(containerName string, containers []v1.Container) string {
 	}
 	return ""
 }
+
+func (pod *Pod) HasIstioSideCar() bool {
+	if len(pod.IstioContainers) > 0 {
```

Comment:
one-liner: `return len(pod.IstioContainers) > 0`. `len(pod.IstioContainers) > 0` is a boolean itself.